app/code/Magento/InventoryConfiguration/Model/SourceItemConfiguration.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryConfiguration\Model;
+
+use Magento\InventoryConfiguration\Api\Data\SourceItemConfigurationInterface;
+use Magento\Framework\Model\AbstractExtensibleModel;
+
+/**
+ * Implementation of basic operations for Source Item Notification entity for specific db layer
+ */
+class SourceItemConfiguration extends AbstractExtensibleModel implements SourceItemConfigurationInterface
```

Comment:
We do not need this class
Because for read and write operations we use `connection`
Also we do not have collection, so for our case `Resource Model` is redundant